### PR TITLE
Added instructions for enabling Runtime Debug Shaders in order to use…

### PR DESCRIPTION
**Pull request by emilybrown1**
In response to a Slack conversation: https://unity.slack.com/archives/GHD5LADU7/p1622619688210400

Ensured the instructions for opening the rendering debugger included the need to enable runtime debug shaders and provided instructions on how to do so.

### CODE REVIEW

**Review by arttu-peltonen — APPROVED** [review]
Good improvement. Mentioning @oleks-k because this part will most likely be duplicated to URP documentation.

**Review by Vic-Cooper — CHANGES_REQUESTED** [review]

File: com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md [hunk 1]
```diff
@@ -16,7 +16,14 @@ The Render Pipeline Debugger window.
...
+To open the Render Pipeline Debugger window in the Editor, you must first enable **Runtime Debug Shaders** in **HDRP Global Settings**:
+
+1. Go to **Edit > Project Settings > Graphics** 
+2. Click **HDRP Settings** and enable **Runtime Debug Shaders**.
+
+To open the Render Pipeline Debugger window, go to **Window > Analysis > Rendering Debugger**. 
+
+You can also open this window at runtime in Play Mode, or in the standalone Unity Player on any device on **Development build**. Use the keyboard shortcut Ctrl+Backspace (Ctrl+Delete on macOS) or press L3 and R3 (Left Stick and Right Stick) on a controller to open the window. You can disable the shortcut through the [enableWindowHotkey variable](https://docs.unity3d.com/Packages/com.unity.render-pipelines.core@12.0/api/UnityEngine.Rendering.DebugManager.html#UnityEngine_Rendering_DebugManager_enableWindowHotkey).
```
Comment (alex-vazquez) [inline com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md:26]: I think you should change the URL version.

```suggestion
You can also open this window at runtime in Play Mode, or in the standalone Unity Player on any device on **Development build**. Use the keyboard shortcut Ctrl+Backspace (Ctrl+Delete on macOS) or press L3 and R3 (Left Stick and Right Stick) on a controller to open the window. You can disable the shortcut through the [enableWindowHotkey variable](https://docs.unity3d.com/Packages/com.unity.render-pipelines.core@latest/api/UnityEngine.Rendering.DebugManager.html#UnityEngine_Rendering_DebugManager_enableWindowHotkey).
```

**Review by alex-vazquez — APPROVED** [review]

File: com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md (hunk 1 above)
Comment (arttu-peltonen) [reply com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md:26]: I'm actually changing this API reference in my PR so I've fixed it there. https://github.com/Unity-Technologies/Graphics/pull/5339/files#diff-4755bb60d17bf206f39d7348502d2f8dfe25ccfe7b24f46a05879c3222f151e4L19

Comment (alex-vazquez) [reply com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md:26]: nice :)

File: com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md [hunk 2]
```diff
@@ -16,7 +16,14 @@ The Render Pipeline Debugger window.
 
 ## Using the Render Pipeline Debugger window
 
-To open the Render Pipeline Debugger window in the Editor, go to  **Window > Analysis > Rendering Debugger**. You can also open this window at runtime in Play Mode, or in the standalone Unity Player on any device on **Development build**. Use the keyboard shortcut Ctrl+Backspace (Ctrl+Delete on macOS) or press L3 and R3 (Left Stick and Right Stick) on a controller to open the window. You can disable the shortcut through the [enableWindowHotkey variable](https://docs.unity3d.com/Packages/com.unity.render-pipelines.core@12.0/api/UnityEngine.Rendering.DebugManager.html#UnityEngine_Rendering_DebugManager_enableWindowHotkey).
+To open the Render Pipeline Debugger window in the Editor, you must first enable **Runtime Debug Shaders** in **HDRP Global Settings**:
+
+1. Go to **Edit > Project Settings > Graphics** 
+2. Click **HDRP Settings** and enable **Runtime Debug Shaders**.
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md:22]: I'd say that "Enable Runtime Debug Shaders" is the third step here.

File: com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md [hunk 3]
```diff
@@ -16,7 +16,14 @@ The Render Pipeline Debugger window.
 
 ## Using the Render Pipeline Debugger window
 
-To open the Render Pipeline Debugger window in the Editor, go to  **Window > Analysis > Rendering Debugger**. You can also open this window at runtime in Play Mode, or in the standalone Unity Player on any device on **Development build**. Use the keyboard shortcut Ctrl+Backspace (Ctrl+Delete on macOS) or press L3 and R3 (Left Stick and Right Stick) on a controller to open the window. You can disable the shortcut through the [enableWindowHotkey variable](https://docs.unity3d.com/Packages/com.unity.render-pipelines.core@12.0/api/UnityEngine.Rendering.DebugManager.html#UnityEngine_Rendering_DebugManager_enableWindowHotkey).
+To open the Render Pipeline Debugger window in the Editor, you must first enable **Runtime Debug Shaders** in **HDRP Global Settings**:
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md:19]: Change "Render Pipeline Debugger window" to "Rendering debugger" in all instances, as per our earlier fixes. This is in an attempt to mitigate merge conflicts.

**Review by Vic-Cooper — CHANGES_REQUESTED** [review]

**Review by Vic-Cooper — APPROVED** [review]
Great! :)

**Comment by sebastienlagarde** [comment]
Removing the need-backport for 10.x as the description don't match anymore what is happening there. If we want a backport we need to do a dedicated PR for 10.x with correct name use in this version. (no global settings, no "rendering debug" etc..

**Pull request merged**